### PR TITLE
IA Pages Index - show coming soon IAs and bring back filters for the dev milestone

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
@russellholt @jagtalon 
Coming Soon IAs are the ones which have the "ready" dev milestone.
Screenshots:
screen shot 2015-02-16 at 16 06 52

(On my instance Airlines has the "ready" dev milestone)
screen shot 2015-02-16 at 16 07 13

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
Updated screenshot (I've just modified a bit the borders of the dev milestone section for consistency):
screen shot 2015-02-16 at 16 35 25

File: lib/DDGC/Web/Controller/InstantAnswer.pm [hunk 1]
```diff
@@ -58,7 +58,10 @@ sub ialist_json :Chained('base') :PathPart('json') :Args() {
 
     my @ial = $rs->search(
         {'topic.name' => { '!=' => 'test' },
-         'me.dev_milestone' => { '=' => 'live'}
+         -or => [
```
Comment (russellholt) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: I can guess what this construct does, but this ` -or ` syntax.. what does it mean?

**Comment by russellholt** [comment]
Looks good!

File: lib/DDGC/Web/Controller/InstantAnswer.pm (hunk 1 above)
Comment (MariagraziaAlastra) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: https://metacpan.org/pod/DBIx::Class::Manual::Cookbook#Complex-WHERE-clauses

**Pull request merged**